opennlp-tools/src/main/java/opennlp/tools/parser/Parse.java

```diff
@@ -319,32 +323,25 @@ public void insert(final Parse constituent) {
       int pn = parts.size();
       for (; pi < pn; pi++) {
         Parse subPart = parts.get(pi);
-        //System.err.println("Parse.insert:con="+constituent+" sp["+pi+"] "+subPart+" "+subPart.getType());
```

Comment:
Maybe we should re-add these statements and replace them with `trace(...)`. If we ever need to debug it, we can switch to trace to see what happens.

Reply:
+1 too. Especially to `trace`. Before since the output of tests & tools was all in `out`/`err` sometimes I would just ignore it as it was too much to parse it (which could possibly hide some errors). Having some like this in a separate level is extremely helpful :pray: :bow: Thanks!

Reply:
Will change to trace :-)